### DISCUSSION TITLE
Multiple currencies per region

### DISCUSSION THREAD

**Discussion by jsphpl** [medusajs/medusa#9936] [Q&A]
I have a use case where customers need to be able to choose their currency independent of their shipping/billing country. As this is a typical feature of many ecommerce solutions, I was surprised to learn that Medusa enforces a 1:1 relationship between regions and currencies.

My questions:

1. What are the reasons for enforcing this restriction?
2. Is there a known workaround to lift this restriction?

**Comment by srindom** [comment]
Could you elaborate more on why you want to be able to shop in different currencies?

**Reply by jsphpl** [reply]
In this specific use case, the merchant ships from Germany and has many customers in Switzerland. Customers from CH living near the border often have their goods shipped to DE. Some of them want to pay in EUR, some in CHF.

Putting DE and CH in different regions does not make sense because if I mask the medusa region as a "currency switcher", they will be surprised why they cannot choose a shipping address in DE if they chose CHF, vice versa. If I let them choose their region first, they will be confused why they cannot choose currency.

Furthermore, DE and CH need to be in the same region because I need to provide the ability to enter a shipping address in DE and billing address in CH.

@srindom can you please explain why region and currency are 1:1 in medusa? Or alternatively, why each country can be part of only one region?

**Reply by jsphpl** [reply]
@srindom could you please provide some background on those decisions and whether or not the medusa team would accept PRs loosening the mentioned restrictions?

**Reply by jsphpl** [reply]
@srindom is there any chance that this issue gets some priority anytime soon?

**Reply by Damcios-s** [reply]
I'd like to have this feature as well. When selling digital products worldwide in a marketplace setup, the vendor might not want to manually add a price in each currecy, so if they price their digital product in EUR, I would like anyone, anywhere to be able to pay in EUR, even if their country natively uses another currency. Right now if a country has a different currency from the currency used to list a product, they cannot buy it, even though pretty much every bank and payment processor let's you pay in a different currency and many people have a second account in EUR or USD anyway.

Seems like an arbitrary decision to lock regions 1:1 with currencies if you ask me.

**Reply by SummerFans** [reply]
> Could you elaborate more on why you want to be able to shop in different currencies?

Asian countries like Japan, South Korea, and Singapore are unlikely to adopt a shared payment currency; transactions must use their own national currencies.
